### PR TITLE
MQCNN fixes

### PR DESCRIPTION
**Pull request by AaronSpieler**
*Description of changes:*
* fixing edge case of prediction length 1
* and some defaults

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/gluonts/model/seq2seq/_forking_estimator.py [hunk 1]
```diff
@@ -131,9 +131,9 @@ def __init__(
         use_feat_static_cat: bool = False,
         cardinality: List[int] = None,
         embedding_dimension: List[int] = None,
-        add_time_feature: bool = True,
-        add_age_feature: bool = True,
-        enable_decoder_dynamic_feature: bool = True,
+        add_time_feature: bool = False,
+        add_age_feature: bool = False,
+        enable_decoder_dynamic_feature: bool = False,
```
Comment (lostella) [inline src/gluonts/model/seq2seq/_forking_estimator.py:136]: Why disabling these by default?

File: src/gluonts/model/seq2seq/_transform.py [hunk 2]
```diff
@@ -171,7 +171,12 @@ def flatmap_transform(
                         ):
                             dec_field[:] = ts[:, idx : idx + self.dec_len].T
 
-                    out[self._future(ts_field)] = np.squeeze(forking_dec_field)
+                    if forking_dec_field.shape[-1] == 1:
```
Comment (lostella) [inline src/gluonts/model/seq2seq/_transform.py:174]: Shape won’t really work with hybridize=True I guess, or I’m missing something?

Comment (lostella) [reply src/gluonts/model/seq2seq/_transform.py:174]: Sorry, I misunderstood: we're in numpyland

Comment (lostella) [reply src/gluonts/model/seq2seq/_transform.py:174]: Wouldn't it make sense to add a test for this?

File: src/gluonts/model/seq2seq/_forking_estimator.py (hunk 1 above)
Comment (AaronSpieler) [reply src/gluonts/model/seq2seq/_forking_estimator.py:136]: So they are by default disabled in MQCNN. This is just to make this consistent.

File: src/gluonts/model/seq2seq/_transform.py (hunk 2 above)
Comment (AaronSpieler) [reply src/gluonts/model/seq2seq/_transform.py:174]: Yeah we probably should. We should add much more edge case tests in general.
